### ISSUE TITLE
Issues when building the environments

### ISSUE THREAD

**Issue by bluecontra** [hyintell/COOM#2]
Hi, 

Thanks a lot for providing this interesting benchmark!

I have some trouble when trying out the environments.

I installed COOM according to the README with `pip install COOM` (the pip way) `git clone https://github.com/hyintell/COOM` (the git clone & pip install way), and encountered the same issues below.

First, I try the example code as below and got "no module named COOM" error.
![image](https://github.com/user-attachments/assets/7a73de77-539f-42b6-ac3d-9414b4c2a8c4)

I verified this by directly import it:
![image](https://github.com/user-attachments/assets/318232e1-7d66-46a9-9bee-d7c93bb87808)

Then, I copied the COOM source code folder provided in this repo to the same path or the script and now the module COOM can be found.
I ran "./COOM/examples/run_single.py" and got a wrapper issue:
> Traceback (most recent call last):
  File "/home/thy/projects/continual_rl_coom/COOM/examples/run_single.py", line 3, in <module>
    from COOM.env.builder import make_env
  File "/home/thy/projects/continual_rl_coom/COOM/env/builder.py", line 4, in <module>
    from gymnasium.wrappers import NormalizeObservation, FrameStack, RecordVideo
ImportError: cannot import name 'FrameStack' from 'gymnasium.wrappers' (/home/thy/anaconda3/envs/coom_pip/lib/python3.9/site-packages/gymnasium/wrappers/__init__.py)

Then I manually checked the code and replaced 'FrameStack' with 'FrameStackObservation', and the error was gone. But another error in `env.reset()` showed up:

> Traceback (most recent call last):
  File "/home/thy/projects/continual_rl_coom/COOM/examples/run_single.py", line 32, in <module>
    main(parser.parse_args())
  File "/home/thy/projects/continual_rl_coom/COOM/examples/run_single.py", line 10, in main
    env.reset()
  File "/home/thy/anaconda3/envs/coom_pip/lib/python3.9/site-packages/gymnasium/wrappers/stateful_observation.py", line 436, in reset
    obs, info = self.env.reset(seed=seed, options=options)
  File "/home/thy/anaconda3/envs/coom_pip/lib/python3.9/site-packages/gymnasium/core.py", line 543, in reset
    obs, info = self.env.reset(seed=seed, options=options)
TypeError: reset() got an unexpected keyword argument 'seed'

I would like to know, do you any idea about this? Did I install COOM in a wrong way like missing dependencies or using incorrect versions of some packages?

**Comment by TTomilin** [comment]
Hi,

Thank you for attempting to use our benchmark! I believe the 2nd and 3rd errors are due to using a newer version of gymnasium. I indeed noticed that we haven't explicitly specified the version of gymnasium in `setup.py`. It is currently installed as a dependency of vizdoom which then most likely uses the most recent version. COOM is compatible with `Version: 0.28.1`. Thank you for reporting this, we will include it in the setup.py.

The reason for not finding the module in the 1st error depends on how the script is executed. When using an IDE, the working directory should be correctly specified. When running from the command line or through some workload managed like slurm, the paths need to be appropriately configured. We don't use any relative imports in the code, so this shouldn't be an issue. This is not something we've encountered ourselves. Let us know if you manage to resolve the issue(s) or experience any further problems.
